var/spack/repos/builtin/packages/draco/package.py

```diff
@@ -25,20 +26,40 @@ class Draco(CMakePackage):
     version('6_20_1', sha256='b1c51000c9557e0818014713fce70d681869c50ed9c4548dcfb2e9219c354ebe')
     version('6_20_0', sha256='a6e3142c1c90b09c4ff8057bfee974369b815122b01d1f7b57888dcb9b1128f6')
 
-    variant('lapack',   default=False, description='Enable LAPACK Wrapper')
+    variant('build_type', default='Release', description='CMake build type',
+            values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
```

Comment:
Is `build_type` part of `CMakePackage`?  If not why not?

Reply:
It is, the presence in this package is simply to change the default to `Release` instead of `RelWithDebInfo`.

Reply:
For our UseCase, we really do want `Release` as the default due to significant performance improvements.

Reply:
@adamjstewart What is the rationale behind changing this default for a single package?  If people want `Release`, they should put it in the `all:` section of `packages.yaml`, and get that behavior across the board.  Changing the default for a single package is just the kind of thing that will trip people up.  ("I set default to `Release` in my `packages.yaml`, why don't I get debug symbols for `draco`???")